apps/client/src/sttr_client/models/question.js

```diff
@@ -30,10 +31,11 @@ class Question {
   // private __type: string;
 
   /**
-   * @param {string} type - data type of the question, eg. 'boolean', or 'geo'
+   * @param {string} type - datatype of the question, eg. 'boolean', or 'geo'
```

Comment:
It's actually data type, or just type ;-)

Reply:
you're right, will fix!